aerospike_enterprise/metadata.csv

```diff
@@ -0,0 +1,651 @@
+aerospike.namespace.batch_sub_lang_read_success,gauge,,,,Number of successful batch-index UDF read subtransactions.,0,aerospike,,,
+aerospike.namespace.batch_sub_lang_write_success,gauge,,,,Number of successful batch-index UDF write subtransactions.,0,aerospike,,,
+aerospike.namespace.batch_sub_proxy_complete,gauge,,,,The number of proxied batch-index sub transactions that completed.,0,aerospike,,,
+aerospike.namespace.batch_sub_proxy_error,gauge,,,,The number of proxied batch-index sub transactions that failed with an error.,0,aerospike,,,
+aerospike.namespace.batch_sub_proxy_timeout,gauge,,,,The number of proxied batch-index sub transactions that timed out.,0,aerospike,,,
+aerospike.namespace.batch_sub_read_error,gauge,,,,The number of batch-index read sub transaction that failed with an error.,0,aerospike,,,
+aerospike.namespace.batch_sub_read_filtered_out,gauge,,transaction,,The number of batch-index read sub transactions that did not happen because the record was filtered out via a predicate expression,0,aerospike,,,
+aerospike.namespace.batch_sub_read_not_found,gauge,,,,The number of batch-index read sub transaction that resulted in not found.,0,aerospike,,,
+aerospike.namespace.batch_sub_read_success,gauge,,,,The number of successful batch-index read sub transactions.,0,aerospike,,,
+aerospike.namespace.batch_sub_read_timeout,gauge,,,,The number of batch-index read sub transactions that timed out.,0,aerospike,,,
+aerospike.namespace.batch_sub_tsvc_error,gauge,,,,The number of batch-index read sub transactions that failed with an error in the transaction service before attempting to handle the transaction.,0,aerospike,,,
+aerospike.namespace.batch_sub_tsvc_timeout,gauge,,,,The number of batch-index read sub transactions that timed out in the transaction service before attempting to handle the transaction.,0,aerospike,,,
+aerospike.namespace.batch_sub_udf_complete,gauge,,,,Number of completed batch-index UDF subtransactions (for scan/query background UDF jobs),0,aerospike,,,
+aerospike.namespace.batch_sub_udf_error,gauge,,,,Number of failed batch-index UDF subtransactions (for scan/query background UDF jobs). Does not include timeouts,0,aerospike,,,
+aerospike.namespace.batch_sub_udf_filtered_out,gauge,,,,Number of batch-index UDF subtransactions that did not happen because the record was filtered out with Filter Expressions.,0,aerospike,,,
+aerospike.namespace.batch_sub_udf_timeout,gauge,,,,Number of batch-index UDF subtransactions that timed out (for scan/query background UDF jobs),0,aerospike,,,
+aerospike.namespace.batch_sub_write_error,gauge,,,,Number of batch-index write subtransactions that failed with an error,0,aerospike,,,
+aerospike.namespace.batch_sub_write_filtered_out,gauge,,,,Number of batch-index write subtransactions that did not happen because the record was filtered out with Filter Expressions,0,aerospike,,,
```

Comment:
```suggestion
aerospike.namespace.batch_sub_udf_timeout,gauge,,,,Number of batch-index UDF subtransactions that timed out (for scan/query background UDF jobs).,0,aerospike,,,
aerospike.namespace.batch_sub_write_error,gauge,,,,Number of batch-index write subtransactions that failed with an error.,0,aerospike,,,
aerospike.namespace.batch_sub_write_filtered_out,gauge,,,,Number of batch-index write subtransactions that did not happen because the record was filtered out with Filter Expressions.,0,aerospike,,,
```